centos_ci.sh

```diff
@@ -30,10 +30,8 @@ yum -y install \
   kvm \
   qemu-kvm \
   libvirt \
-  python2-pip
-
-# Install 'requests' module used by tests/utils/minishift_latest_version.py
-pip install requests
+  python2-pip \
```

Comment:
Do we really need  pip now as we are installing python-requests through YUM.

Reply:
yes you are right